app/layout/project/navigation/index.tsx

```diff
@@ -88,9 +96,41 @@ export const Navigation = (): JSX.Element => {
   }, []);
 
   const handleRunScenario = useCallback(() => {
-    // todo: define run scenario button behaviour
-    console.log('handleRunScenario', sid);
-  }, [sid]);
+    setScenarioRunning(true);
+
+    runScenarioMutation.mutate(
+      { id: `${sid}` },
+      {
+        onSuccess: ({ data: { data: s } }) => {
+          setScenarioRunning(false);
+          addToast(
+            'run-start',
+            <>
+              <h2 className="font-medium">Success!</h2>
+              <p className="text-sm">Run started</p>
+            </>,
+            {
+              level: 'success',
+            }
+          );
+          console.info('Scenario runned succesfully', s);
```

Comment:
Also, I know this is a copy and paste from the _Parameters_ tab, but let's try to avoid console things.